proto/umee/incentive/v1/tx.proto

```diff
@@ -0,0 +1,134 @@
+syntax = "proto3";
+package umeenetwork.umee.incentive.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "umee/incentive/v1/incentive.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/incentive";
+
+// Msg defines the x/incentive module's Msg service.
+service Msg {
+  // Claim defines a method for claiming any pending incentive rewards.
+  rpc Claim(MsgClaim) returns (MsgClaimResponse);
+
+  // Bond defines a method for bonding uToken collateral into reward tier.
+  rpc Bond(MsgBond) returns (MsgBondResponse);
+
+  // BeginUnbonding defines a method for starting to unbond uToken collateral.
+  rpc BeginUnbonding(MsgBeginUnbonding) returns (MsgBeginUnbondingResponse);
+
+  // Sponsor defines a permissionless method for sponsoring an upcoming, not yet funded incentive program.
+  rpc Sponsor(MsgSponsor) returns (MsgSponsorResponse);
```

Comment:
I think w should enable partial sponsoring, when 2 separate sponsors will like to sponsor a program. Or do it in v2.

Reply:
Yes, but maybe we need to support a scenario when, for example we have 2 sponsors?

Reply:
Something that could be useful in the future, but probably not near-term. Possible use case would be ecosystems with multiple tokens like Comdex. Maybe they want to incentivize borrowing their CMST stablecoin with CMDX, HARBOR, etc.

Reply:
yes, I was thinking about similar scenario. Other would be if there are two independent accounts which would sponsor only if the proposal passes (Eg: Umee DAO and Osmosis DAO).